File: ProbarFirebase/app/src/main/java/com/example/probarfirebase/Grupo.java
package com.example.probarfirebase;

public class Grupo {
    private String idGrupo;
    private String nombre;

    public Grupo(String idGrupo, String nombre) {
        this.idGrupo = idGrupo;
        this.nombre = nombre;
    }

    public String getIdGrupo() {
        return idGrupo;
    }

    public void setIdGrupo(String idGrupo) {
        this.idGrupo = idGrupo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: PabloMaxia_Examen_Room/app/src/main/java/com/example/pablomaxia_examen_room/modelo/entidades/Grupo.kt
package com.example.pablomaxia_examen_room.modelo.entidades

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.example.pablomaxia_examen_room.modelo.Constantes

@Entity(tableName = Constantes.NOMBRE_TABLA_GRUPO)
data class Grupo(
    @PrimaryKey
    @ColumnInfo(name = "id_grupo")
    val id: String, val nombre: String, val aula: String
)
